Content.Server/Cargo/Systems/CargoSystem.Orders.cs

```diff
@@ -85,9 +85,11 @@ private void UpdateConsole(float frameTime)
             {
                 _timer -= Delay;
 
-                foreach (var account in EntityQuery<StationBankAccountComponent>())
+                var stationQuery = EntityQueryEnumerator<StationBankAccountComponent>();
```

Comment:
pro

Reply:
pro read description, it will work without this but the balance only updates after you close and re-open the UI
https://github.com/space-wizards/space-station-14/pull/33123